docs/management_api.md

```diff
@@ -15,18 +15,18 @@ Similar as [Inference API](inference_api.md), Management API also provide a [API
 
 ### Register a model
 
-* POST /models
-** model_url - Model archive download url. support:
-*** local model archive (.mar), the file must be directly in model_store folder.
-*** local model directory, the directory must be directly in model_store foler. This option can avoid MMS extracting .mar file to temporary folder, which will improve load time and reduce disk space usage.
-*** HTTP(s) protocol. MMS can download .mar files from internet.
-** model_name - Name of the model. This name will be used as {model_name} in other API as path. If this parameter is not present, modelName in MANIFEST.json will be used.
-** handler - Inference handler entry-point. This value will override handler in MANIFEST.json if present.
-** runtime - Runtime for the model custom service code. This value will override runtime in MANIFEST.json if present. Default PYTHON.
-** batch_size - Inference batch size, default: 1.
-** max_batch_delay - Maximum delay for batch aggregation, default: 100 millisecnonds.
-** initial_worker - Number of initial workers to create, default: 0.",
-** synchronous - Decides whether creation of worker synchronous or not, default: false.
+`POST /models`
+* model_url - Model archive download url. support:
+    * local model archive (.mar), the file must be directly in model_store folder.
+    * local model directory, the directory must be directly in model_store foler. This option can avoid MMS extracting .mar file to temporary folder, which will improve load time and reduce disk space usage.
+    * HTTP(s) protocol. MMS can download .mar files from internet.
+* model_name - Name of the model. This name will be used as {model_name} in other API as path. If this parameter is not present, modelName in MANIFEST.json will be used.
+* handler - Inference handler entry-point. This value will override handler in MANIFEST.json if present.
```

Comment:
```suggestion
* handler - the inference handler entry-point. This value will override `handler` in MANIFEST.json if present.
```

Reply:
The phrase "inference handler entry-point" isn't really clear. Is this `predictions`?

Reply:
the handler referring to custom service code, in the past user has to extends from MXNetBaseService.py, now it can be arbitrary python function. This function is the custom service code's entry-point.